app/Notifications/TwitterNotSent.php

```diff
@@ -5,73 +5,73 @@
 use App\Exceptions\ShouldDeleteNotificationException;
 use App\Models\Status;
 use Illuminate\Bus\Queueable;
-use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Notifications\DatabaseNotification;
 use Illuminate\Notifications\Notification;
-use Illuminate\Notifications\Messages\MailMessage;
+use JetBrains\PhpStorm\ArrayShape;
 
 class TwitterNotSent extends Notification
 {
     use Queueable;
 
-    public $error;
-    public $status;
+    public string $error;
+    public Status $status;
 
-    public function __construct($error, Status $status)
-    {
+    public function __construct(string $error, Status $status) {
         $this->error  = $error;
         $this->status = $status;
     }
 
-    public function via()
-    {
+    public function via(): array {
         return ['database'];
     }
 
-    public function toArray()
-    {
+    #[ArrayShape(['error' => "string", 'status_id' => "int"])]
+    public function toArray(): array {
         return [
-            'error' => $this->error,
+            'error'     => $this->error,
             'status_id' => $this->status->id,
         ];
     }
 
-    public static function detail($notification)
-    {
+    /**
+     * @param $notification
+     * @return string
+     * @throws ShouldDeleteNotificationException
+     */
+    public static function detail(DatabaseNotification $notification): string {
         $data                 = $notification->data;
         $notification->detail = new \stdClass();
 
         try {
             $status = Status::findOrFail($data['status_id']);
-        } catch(ModelNotFoundException $e) {
+        } catch (ModelNotFoundException) {
             throw new ShouldDeleteNotificationException();
         }
 
         $notification->detail->status = $status;
         return $notification->type;
     }
 
-    public static function render($notification)
-    {
+    public static function render(DatabaseNotification $notification): ?string {
         try {
-            $detail = Self::detail($notification);
-        } catch (ShouldDeleteNotificationException $e) {
+            self::detail($notification);
+        } catch (ShouldDeleteNotificationException) {
             $notification->delete();
             return null;
         }
 
         $data = $notification->data;
-
         return view("includes.notification", [
-            'color' => "warning",
-            'icon' => "fas fa-exclamation-triangle",
-            'lead' => __('notifications.socialNotShared.lead', ['platform' => "Twitter"]),
-            "link" => route('statuses.get', ['id' => $detail->status->id]),
+            'color'  => "warning",
+            'icon'   => "fas fa-exclamation-triangle",
+            'lead'   => __('notifications.socialNotShared.lead', ['platform' => "Twitter"]),
+            "link"   => route('statuses.get', ['id' => $data['status_id']]),
```

Comment:
This line is the whole change. We have tried to access a variable that does not exist.